data/Nigeria/Senate/sources/instructions.json

```diff
@@ -0,0 +1,34 @@
+{
+  "sources": [
+    {
+      "file": "morph/data.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/nigeria-national-assembly",
+        "query": "SELECT * FROM data WHERE js_position = 'Sen'"
+      },
+      "source": "http://www.nass.gov.ng/",
+      "type": "membership"
+    },
+    {
+      "file": "manual/terms.csv",
+      "type": "term"
+    },
+    {
+      "file": "wikidata/groups.json",
+      "type": "group",
+      "create": {
+        "from": "group-wikidata",
+        "source": "manual/group_wikidata.csv"
+      }
+    },
+    {
+      "file": "wikidata/elections.json",
+      "type": "wikidata-elections",
+      "create": {
+        "from": "election-wikidata",
+        "base": "Q22283753"
```

Comment:
I can't find a better Wikidata item for this (there doesn't seem to be one just for senatorial elections in Nigeria), so I've removed this source.